### PR TITLE
Ps migrations epmrpp 34483

### PR DESCRIPTION
**Pull request by dshybeka**
- Added docker compose for service-api
- Externalized rebbitmq, postgre and datastore properties

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -47,17 +47,17 @@ rp.email.debug=false
 rp.email.account=
 rp.email.password=
 
+spring.application.name=api
+info.build.version=4.0n
```
Comment (avarabyeu) [inline src/main/resources/application.properties]: info.build.version=${rp.api.version:4.0}

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -47,17 +47,17 @@ rp.email.debug=false
...
+
 #show sql statement
 logging.level.org.hibernate.SQL=debug
 
 #show sql values
 logging.level.org.hibernate.type.descriptor.sql=trace
 
-spring.rabbitmq.host=localhost
-spring.rabbitmq.port=5672
+spring.rabbitmq.host=${RABBIT_HOST:localhost}
```
Comment (avarabyeu) [inline src/main/resources/application.properties]: +spring.rabbitmq.host=${rp.rabbit.host:localhost}

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -103,3 +98,9 @@ springfox:
     swagger:
       v2:
         path: /api-docs
+
+
+spring.datasource.driver-class-name: org.postgresql.Driver
+spring.datasource.username: rpuser
+spring.datasource.password: rppass
+spring.datasource.url: ${DATASOURCE_URL:jdbc:postgresql://localhost:5432/reportportal}
```
Comment (avarabyeu) [inline src/main/resources/application.yaml]: rp.db.url

File: src/main/resources/datastore.properties [hunk 4]
```diff
@@ -0,0 +1,5 @@
+datastore.default.path=${DATASTORE_DEFAULT_PATH:/data/storage}
+datastore.seaweed.master.host=${DATASTORE_MASTER_HOST:localhost}
+datastore.seaweed.master.port=${DATASTORE_MASTER_PORT:9333}
+# could be one of [seaweed, filesystem]
+datastore.type=${DATASTORE_TYPE:filesystem}
```
Comment (avarabyeu) [inline src/main/resources/datastore.properties]: rp.binarystore

Comment (dshybeka) [reply src/main/resources/datastore.properties]: updated

File: src/main/resources/application.properties (hunk 1 above)
Comment (dshybeka) [reply src/main/resources/application.properties]: updated

File: src/main/resources/application.properties (hunk 2 above)
Comment (dshybeka) [reply src/main/resources/application.properties]: updated

File: src/main/resources/application.yaml (hunk 3 above)
Comment (dshybeka) [reply src/main/resources/application.yaml]: updated

**Pull request merged**